supplementary_style_guide/style_guidelines/specific-doc-types.adoc

```diff
@@ -5,5 +5,104 @@
 [[release-notes]]
 == Release notes
 
+[[release-notes-doc-texts]]
+=== Conventions for phrasing Release Notes doc texts
```

Comment:
This whole guide is a collection of conventions, so I think we could just call this entry "Release notes doc texts"

I don't know if "doc texts" is an official/accepted term, but it does sound weird to me. But if others are okay with it, I'm fine with it as well, since we define the term here.